src/calendars/chile.js

```diff
@@ -52,13 +52,13 @@ let dates = year => {
     // Commented because the key and rank is the same as the definition in general.js
```

Comment:
I believe the problem lays in the addition of Optional Memorial of St Henry, Bishop of Martyr ([src](https://en.wikipedia.org/wiki/General_Roman_Calendar#Chile)), as on 13 July (when St Henry is celebration in `general.js`) in `chile.js` is Saint Teresa of Los Andes, Virgin (Feast) is celebrated.

I think you should correct this in this PR (and then remove the comment on line 52).

Reply:
I hadn't even seen it. What should we do here, remove all this commented block?

Reply:
You should do this:
1. Remote that comment block.
2. Create a new key (localise it in `en.js`): `"saintCamillusDeLellisPriestOrSaintHenryBishopAndMartyr": "Saint Camillus de Lellis, priest, or Saint Henry, Bishop and Martyr"` (`sk.js`: `Svätého Kamila de Lellis, kňaza, alebo svätého Henricha`).
3. Define `saintCamillusDeLellisPriestOrSaintHenryBishopAndMartyr` celebration in `chile.js`.

Reply:
Ok, this isn't very related to this PR, but I can do it as a separated commit.

Reply:
Moved to issue #116.